cmd/ike/session/session.go

```diff
@@ -26,13 +22,23 @@ func Offline(cmd *cobra.Command) (func(), error) {
 	return func() {}, nil
 }
 
+var sessionClient *client
+
 // CreateOrJoinHandler provides the option to either create a new session if non exist or join an existing.
 // Rely on the following flags:
 //  * deployment - the name of the target deployment and will update the flag with the new deployment name
 //  * session - the name of the session
 func CreateOrJoinHandler(cmd *cobra.Command) (func(), error) {
 	sessionName := getSessionName(cmd)
 	deploymentName, _ := cmd.Flags().GetString("deployment")
+	if sessionClient == nil {
```

Comment:
Not particularly happy with this "lazy loading" but couldn't come up with a better approach (besides wrapping it up in a `func` and use that instead of `var` reference in all the `func`s here). Any suggestions on how we could make it better?

Reply:
Could move the instance to where it's created like with http.DefaultClient type of thing.

Move sessionClient to session_client and lazy it in there. Separate between DefaultClient() and NewClient() or similar. Basically the same, but less scattered.

Reply:
Done 711872a